stoplightio/json-schema-viewer
# fix(json-schema-viewer): Description for oneOf/anyOf does not display

Addresses #7465

**Summary:**

When selecting `oneOf` or `anyOf` and providing a description, the description fails to show. There are several cases where the description will not show:

1. If there are more than one object
2. If there is an empty string on the object's description

I have provided a video to explain these bugs:

https://user-images.githubusercontent.com/52048026/143163552-ca5d9853-29cd-4e13-ae64-cfee4b65f1f7.mov

**To Test:**

1. Check out the branch and run `yarn storybook`
2. I have provided a video and all use-cases for these bugs.


https://user-images.githubusercontent.com/52048026/143164290-08e1e6a5-bca0-43ae-b0eb-9a3530311a69.mp4


Changing one line of code fixed this issue, but I'm unclear as to why `schemaNode.annotations.descriptions` was used instead of `schemaNode.originalFragment.descriptions`

**Checklist**

- The basics
  - [x] I tested these changes manually in my local or dev environment
- Tests
  - [ ] Added or updated
- Event Tracking
  - [ ] I added event tracking and followed the event tracking guidelines
- Error Reporting
  - [ ] I reported errors and followed the error reporting guidelines
